### ISSUE TITLE
VoiceOver not reading selection using PgUp / PgDn in command palette

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6881]
### System details

    RStudio Edition : Server
    RStudio Version : 1.4.344
    OS Version      : macOS / Safari / VoiceOver
    R Version       : Any

### Steps to reproduce the problem
- connect to RStudio Server using Safari and VoiceOver
- <kbd>Ctrl+Shift+P</kbd> to invoke command palette
- use up/down arrows to change selection, selection is read by VoiceOver
- use PgUp or PgDn keys to change selection

### Describe the problem in detail
- selection not ready by VoiceOver

### Describe the behavior you expected
- navigating with PgUp / PgDn should announce selected items
- works with NVDA or JAWS on Chrome for Windows
